.cursorrules

```diff
@@ -84,6 +84,31 @@ Documentation:
 - Use JSDoc and TypeDoc for public APIs.
 - Include examples and document error cases.
 
+Changelog management:
+
+- Follow Keep a Changelog format for all CHANGELOG.md files.
+- For release PRs, categorize "### Uncategorized" entries into:
+  - "### Added" for new features and functionality (entries starting with "feat:")
+  - "### Changed" for changes to existing functionality (entries starting with "chore:" or breaking changes)
+  - "### Removed" for removed functionality or features
+  - "### Fixed" for bug fixes and corrections (entries starting with "fix:" or "refactor:")
```

Comment:
I should've mentioned this earlier, but we should complete this section with the other section titles from: https://keepachangelog.com/en/1.0.0/#how

Reply:
Maybe even mention "keep a changelog v1" in these instructions.